library/components/TrendArrow.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <div class="trendArrow">
+        <arrow-up-icon v-if="difference > 0" :size="iconSize" />
+        <minus-icon v-else-if="difference === 0" :size="iconSize" />
+        <arrow-down-icon v-else :size="iconSize" />
```

Comment:
If you wanted you could load the 'smarts' about the first and last number and calculating the difference into CurrentAndPast. This component could take a prop like

```
    type: {
      validator: function (value) {
        // The value must match one of these strings
        return ['up', 'down', 'neutral'].includes(value)
      }
```

thise would avoid duplicating the 'first' and 'second' number that the  `CurrentAndPast` has and we could expand it relatively easily... As well as use it without numbers... Feel free to go with the direction you like though.

Reply:
Reducing the input to a single value is a fantastic idea! I think having it stay a number makes sense from an ease of use standpoint, but there is no reason a user can't do the difference calculation themselves which would neatly sidestep the bad prop names issue. :D

Reply:
Agreed with the above. Additionally there is a non-trivial concern that users will see up/neutral/down as a prop and assume that those are the _only_ valid values.